pages/[lng]/categories/index.tsx

```diff
@@ -0,0 +1,151 @@
+  openOrderLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__openorder}`,
+  preOrderLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__preorder}`,
+  newLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__new}`,
+  productImageContainerClassName: styleProduct.product_link,
+  productImageClassName: styleProduct.product_itemContainerImage,
+  productLabelContainerClassName: styleProduct.product_label,
+  productTitleClassName: styleProduct.product_label__title,
+  productPriceClassName: styleProduct.product_labelPrice,
+  salePriceClassName: styleProduct.product_labelPrice__sale,
+  priceClassName: styleProduct.product_labelPrice__price,
+  productsListCategoryHeaderTitleClassName: styles.productsListCategoryHeaderTitle,
+  productsListCategoryHeaderClassName: styles.productsListCategoryHeader,
+  productsListCategoryContainerClassName: styles.productsListCategoryContainer,
+  productsListCategoryHeaderLinkClassName: styles.productsListCategoryHeaderLink,
+  productsListContainerClassName: styles.productsListContainer,
+}
+
+const CategoriesPage: FC<any> = ({
+  lng,
+  lngDict,
+  brand,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const router = useRouter()
+  const i18n: any = useI18n()
+  const size = useWindowSize()
+
+  const [pageInfo, setPageInfo] = useState({
+    pageNumber: 0,
+    itemPerPage: 6,
+    totalItems: 0,
+  })
+
+  const { currPage } = useInfiniteScroll(pageInfo, 'category_list:last-child')
+
+  const layoutProps = {
+    i18n,
+    lng,
+    lngDict,
+    brand,
+    headerTitle: i18n.t('categories.title'),
+    SEO: {
+      title: i18n.t("categories.title")
+    }
+  }
+
+  return (
+    <Layout {...layoutProps}>
+      <SEO title={i18n.t("product.products")} />
+      <div className="container mt-5 pt-4 pb-3">
+        <Breadcrumb
+          steps={[{ label: i18n.t('breadcrumb.home') }, { label: i18n.t('categories.categories') }]}
+        />
+      </div>
+      <div className={`container ${styleProducts.products}`}>
+        <div className="row">
+          {Array.from(Array(currPage + 1)).map((_, i) => (
+            <ProductListByCategory
+              key={i}
+              pageNumber={i}
+              productCategoryType="INFINITE_SCROLL"
+              itemPerPage={pageInfo.itemPerPage}
+              classes={classesProductCategory}
+              itemProductsPerCategory={3}
+              getPageInfo={setPageInfo as any}
+              thumborSetting={{
+                width: size.width < 768 ? 512 : 800,
+                format: "webp",
+                quality: 85,
+              }}
+              emptyStateComponent={
+                <div className="col-12 my-3">
+                  <EmptyComponent
+                    icon={<RiQuestionFill color="#A8A8A8" size={20} />}
```

Comment:
Kayaknya di figma pake nya react-icons/io5 IoHelpCircle sama warnanya "#BCBCBC". 
```suggestion
                    icon={<IoHelpCircle color="#BCBCBC" size={20} />}
```

Reply:
Oiya jangan lupa importnya diganti juga ya